api/v1alpha1/virtualmachineimage_types.go

```diff
@@ -41,6 +41,21 @@ type VirtualMachineImageOSInfo struct {
 	Type string `json:"type,omitempty"`
 }
 
+
+// VirtualMachineImageOVFEnv describes information related to a user configurable property element
+// that is supported by VirtualMachineImage and can be customized during VirtualMachine deployment.
```

Comment:
Don't use "deployment" because that means something very different in kubernetes

Reply:
Got it. I will fix this in another MR.